apps/drec-ui/src/shared/components/ConnectMetamask/ConnectMetamask.tsx

```diff
@@ -0,0 +1,59 @@
+import { NotificationTypeEnum, showNotification } from '@energyweb/origin-ui-core';
+import { Button, Box, Typography, useMediaQuery, Paper } from '@material-ui/core';
+import { useTheme } from '@material-ui/core';
+import { UnsupportedChainIdError, useWeb3React } from '@web3-react/core';
+import React from 'react';
+import { injectedConnector } from '../../../utils';
```

Comment:
you can use `utils` import. Also you don't need React import because cra config allows to skip this import